R/epinow.R

```diff
@@ -445,17 +454,19 @@ regional_epinow <- function(reported_cases,
                                               future.scheduling = Inf)
 
   # names on regional_out
-  names(regional_out) <- regions # ["foo" => a, "bar" => b$error, "baz" => c, "parrot" => d$error]
+  names(regional_out) <- regions
```

Comment:
sorry, I typed this talking a colleague through a problem I was having and managed to leave it in!

Reply:
no problem!